thirdeye-detectionpipeline/src/main/java/ai/startree/thirdeye/detectionpipeline/DetectionPipelineContext.java

```diff
@@ -27,6 +28,7 @@ public class DetectionPipelineContext {
   private EnumerationItemDTO enumerationItem;
   private DetectionPipelineUsage usage;
   private Long alertId;
+  private AuthorizationConfigurationDTO anomalyAuth;
```

Comment:
Also we should have `alert` here if we are going to do it this way. 
What I don't like is the modularity of the detection pipeline is being broken if we are introducing db level operations in the detection pipeline.
@cyrilou242 @jacksonargo

Reply:
agree, seems to be fixed in the newest commits